acceptance/bundle/validate/empty_resources/with_permissions/output.txt

```diff
@@ -33,11 +36,10 @@ Error: model rname is not defined
 }
 
 === resources.experiments.rname ===
-Error: experiment rname is not defined
-
 {
   "experiments": {
     "rname": {
+      "name": ".",
```

Comment:
I don't know if it makes sense for experiment to have name="." @pietern @lennartkats-db

Related: https://github.com/databricks/cli/pull/1490

Reply:
This is likely because of a `path.Clean` call. The name must be a valid filename (or path, don't recall).